lib/datadog/lambda.rb

```diff
@@ -14,6 +14,10 @@
 require 'json'
 require 'time'
 
+# rubocop:disable Style/GlobalVars
+$IS_COLD_START = true
```

Comment:
Is there any way we could not use a global here? Maybe we could scope the variable to the module instead?

Reply:
I didn't think that would work, but it looks like it does!